crates/but-workspace/tests/fixtures/scenario/insert-blank-commit-three-commits copy.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+echo "/remote/" > .gitignore
+mkdir remote
+pushd remote
+git init
+popd
+
+git init
+
+git remote add origin ./remote
```

Comment:
I don't understand why there is a remote at all, it shouldn't be relevant.
If this is not copy-paste, I think it should be explained.